ml_inference_offloading/src/main/java/ai/nnstreamer/ml/inference/offloading/MainActivity.kt

```diff
@@ -134,9 +134,9 @@ class MainActivity : ComponentActivity() {
 
         val mobileNet = File("$path/mobilenet_v1_1.0_224_quant.tflite")
         val mobileNetFilter =
-            "other/tensor,format=static,dimension=(string)3:224:224:1,type=uint8,framerate=0/1 ! " +
+            "other/tensors,num_tensors=1,format=static,dimension=(string)3:224:224:1,type=uint8,framerate=0/1 ! " +
```

Comment:
dimension ==> dimensions
type ==> types

Reply:
Oh I forgot to change dimension and type.
I will also update nnstreamer-example's Tizen web application too!
Thank you :)

Reply:
You already fixed Tizen web example 😂
Updated android part!